src/g_game.c

```diff
@@ -1168,11 +1168,8 @@ boolean G_Responder(event_t* ev)
   }
 
   if (dclick_use && ev->type == ev_mouseb_down &&
-       (
-         M_InputMatchMouseB(input_strafe, ev->data1) ||
-         M_InputMatchMouseB(input_forward, ev->data1)
-       ) &&
-       ev->data2 >= 2 && (ev->data2 % 2) == 0)
+      (M_InputActivated(input_strafe) || M_InputActivated(input_forward)) &&
```

Comment:
Hm, does this now mean that double-clicking will work with *any* mouse button as long as a strafe or forward input is activated - even by a different device, e.g. keyboard?

Reply:
Above, there is a `ev->type == ev_mouseb_down` check that ensures the mouse button is pressed. I removed `M_InputMatchMouseB` because that's the only place it's used now. The `M_InputActivated` function just checks for the current event (see `M_InputTrackEvent` in `m_input.c`)

Reply:
Okay, that's good to know. Thank you!